sdk/src/assertions/metadata.rs

```diff
@@ -198,7 +198,7 @@ pub struct Actor {
     #[serde(skip_serializing_if = "Option::is_none")]
     pub identifier: Option<String>,
 
-    /// List of [`HashedUri`] references to W3C Verifiable Credentials.
+    /// List of references to W3C Verifiable Credentials.
```

Comment:
Why remove this?

Reply:
Because HashedUri isn't exposed to users. They have no way to manage them. I've got a follow on PR that does more to clean this up. But this reference causes the docs to not generate.